diff --git a/node_modules/discord-rpc/src/transports/ipc.js b/node_modules/discord-rpc/src/transports/ipc.js
index f050a01..4c1ba38 100644
--- a/node_modules/discord-rpc/src/transports/ipc.js
+++ b/node_modules/discord-rpc/src/transports/ipc.js
@@ -15,7 +15,7 @@ const OPCodes = {
 
 function getIPCPath(id) {
   if (process.platform === 'win32') {
-    return `\\\\?\\pipe\\discord-ipc-${id}`;
+    return `\\\\.\\pipe\\discord-ipc-${id}`;
   }
   const { env: { XDG_RUNTIME_DIR, TMPDIR, TMP, TEMP } } = process;
   const prefix = XDG_RUNTIME_DIR || TMPDIR || TMP || TEMP || '/tmp';
